### DISCUSSION TITLE
Create an undyne the undying fight simulator

### DISCUSSION THREAD

**Discussion by UndertaleBossSimulators** [Jcw87/c2-sans-fight#103] [Ideas]
Please make an undyne the undying fight simulator, I can't find any accurate simulators for undyne the undying and I want to fight undyne the undying again.

**Comment by Jcw87** [comment]
Not gonna happen. I have no plans to make other Undertale battle simulators. Even if I were open to that, you are requesting this at one of the worst possible times. I'll be getting access to S&box in a few days, and my dev time will likely be spent on that for the foreseeable future.
